packages/perseus/src/widgets/passage-ref.jsx

```diff
@@ -86,10 +97,9 @@ class PassageRef extends React.Component<Props, State> {
         }
 
         let summaryOutput;
-        if (this.props.summaryText) {
-            const summaryTree = PerseusMarkdown.parseInline(
-                this.props.summaryText,
-            );
+        const summaryText: string = this.props.summaryText;
+        if (summaryText) {
+            const summaryTree = PerseusMarkdown.parseInline(summaryText);
```

Comment:
```suggestion
        if (this.props.summaryText) {
            const summaryTree = PerseusMarkdown.parseInline(
                this.props.summaryText,
            );
```

I checked locally and `summaryText` is already typed as a string, we don't need to retype it. I'd recommend reverting this change.

Reply:
Change is reverted!